libs/features/advert-page/src/lib/features-advert-page.tsx

```diff
@@ -1,102 +1,37 @@
-let description =
-  'Norite dirbti, bet neuždirbti? Jei taip, tuomet šis skelbimas kaip tik Jums! Vergovė vedama patyrusio lektoriaus Donato Kuskio išmokins Jus, kad naudojant NX norisi keiktis. Manote, kad TypeScript tai tikra palaima? Prisijungę prie projekto pamatyse, jog gyvenime taip stipriai dar neklydote. Gyvenimo prasmės netekimas ir motyvacijos praradimas garantuotas!';
-let creator = 'Donatas Kuskys';
-let image =
-  'https://c402277.ssl.cf1.rackcdn.com/photos/14875/images/magazine_hero/Medium_WW248926.jpg?1513623521';
-let categories = [
-  { title: 'Programming', value: 'prog' },
-  { title: 'Javascript', value: 'js' },
-  { title: 'React', value: 'react' },
-  { title: 'Web Development', value: 'web' },
-];
-let onGetStartedClick = () => {
-  alert('paspaudei');
-};
-let lessons = [
-  {
-    title: 'Gecko',
-    authorName: 'Donatas',
-    image: 'https://www.eurekalert.org/multimedia/pub/web/229622_web.jpg',
-  },
-  {
-    title: 'Gecko',
-    authorName: 'Donatas',
-    image: 'https://www.eurekalert.org/multimedia/pub/web/229622_web.jpg',
-  },
-  {
-    title: 'Gecko',
-    authorName: 'Donatas',
-    image: 'https://www.eurekalert.org/multimedia/pub/web/229622_web.jpg',
-  },
-  {
-    title: 'Gecko',
-    authorName: 'Donatas',
-    image: 'https://www.eurekalert.org/multimedia/pub/web/229622_web.jpg',
-  },
-  {
-    title: 'Gecko',
-    authorName: 'Donatas',
-    image: 'https://www.eurekalert.org/multimedia/pub/web/229622_web.jpg',
-  },
-  {
-    title: 'Gecko',
-    authorName: 'Donatas',
-    image: 'https://www.eurekalert.org/multimedia/pub/web/229622_web.jpg',
-  },
-  {
-    title: 'Gecko',
-    authorName: 'Donatas',
-    image: 'https://www.eurekalert.org/multimedia/pub/web/229622_web.jpg',
-  },
-];
-
-let lessonsDescription =
-  'Are you ready to find out what all the hype is about with ReactJS? These ReactJS for beginners tutorials will bring you completely up to speed on the hottest JavaScript framework used on the web today.';
+import { useParams } from 'react-router-dom';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import { useAdverts } from '@ppm/hooks/use-adverts';
 
+import './features-advert-page.scss';
 
-export const FeaturesAdvertPage = (props: FeaturesAdvertPageProps) => {
+export const FeaturesAdvertPage = () => {
   const { id } = useParams();
 
-  useEffect(()=> {
-    /*TODO: load info with given ID from redux*/
-    console.log(id);
-  });
+  const { advert, loading, onGetStartedClick } = useAdverts(id);
+
+  if (loading || !advert) return <CircularProgress />;
 
   return (
     <div>
       <SharedAdvertInfo
-        title={title}
-        description={description}
-        creator={creator}
-        image={image}
-        categories={categories}
+        title={advert.title}
+        description={advert.description}
+        creator={advert.creator.name}
+        image={advert.imageUrl}
+        categories={[{ title: advert.category, value: advert.category }]}
```

Comment:
make function who makes formatted data for this component for ex: advert.category.map((item, index)=> ({title: item.categorie, value: index}))

Reply:
Cannot do "map" function for a string advert.category.